main.go

```diff
@@ -149,6 +153,8 @@ func initOrg() *orgservice.Org {
 		os.Getenv("AUTH0_CLIENT_SECRET"),
 		os.Getenv("AUTH0_AUDIENCE"),
 		os.Getenv("AUTH0_TOKEN_ENDPOINT"),
+		"",
```

Comment:
If you added this new SLACK_WEBHOOK_URL, why didn't you pass it here?

Reply:
There are 2 new vars - one is the secret and one is slack url which is below this line..
As per discussion with @enkhalifapro, secret can be kept empty.

Reply:
Even more, it mt be removed because it was duplicated param.